<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>2 Project summary | fecalFACS Notebook</title>
  <meta name="description" content="A diary-style notebook for this project" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="2 Project summary | fecalFACS Notebook" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A diary-style notebook for this project" />
  <meta name="github-repo" content="https://github.com/GiselaHKopp/fecalFACS" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="2 Project summary | fecalFACS Notebook" />
  
  <meta name="twitter:description" content="A diary-style notebook for this project" />
  

<meta name="author" content="Gisela H. Kopp" />


<meta name="date" content="2022-06-09" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="project-notes.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>

<script>
/* ========================================================================
 * Bootstrap: transition.js v3.3.7
 * http://getbootstrap.com/javascript/#transitions
 * ========================================================================
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 * ======================================================================== */


+function ($) {
  'use strict';

  // CSS TRANSITION SUPPORT (Shoutout: http://www.modernizr.com/)
  // ============================================================

  function transitionEnd() {
    var el = document.createElement('bootstrap')

    var transEndEventNames = {
      WebkitTransition : 'webkitTransitionEnd',
      MozTransition    : 'transitionend',
      OTransition      : 'oTransitionEnd otransitionend',
      transition       : 'transitionend'
    }

    for (var name in transEndEventNames) {
      if (el.style[name] !== undefined) {
        return { end: transEndEventNames[name] }
      }
    }

    return false // explicit for ie8 (  ._.)
  }

  // http://blog.alexmaccaw.com/css-transitions
  $.fn.emulateTransitionEnd = function (duration) {
    var called = false
    var $el = this
    $(this).one('bsTransitionEnd', function () { called = true })
    var callback = function () { if (!called) $($el).trigger($.support.transition.end) }
    setTimeout(callback, duration)
    return this
  }

  $(function () {
    $.support.transition = transitionEnd()

    if (!$.support.transition) return

    $.event.special.bsTransitionEnd = {
      bindType: $.support.transition.end,
      delegateType: $.support.transition.end,
      handle: function (e) {
        if ($(e.target).is(this)) return e.handleObj.handler.apply(this, arguments)
      }
    }
  })

}(jQuery);
</script>
<script>
/* ========================================================================
 * Bootstrap: collapse.js v3.3.7
 * http://getbootstrap.com/javascript/#collapse
 * ========================================================================
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 * ======================================================================== */

/* jshint latedef: false */

+function ($) {
  'use strict';

  // COLLAPSE PUBLIC CLASS DEFINITION
  // ================================

  var Collapse = function (element, options) {
    this.$element      = $(element)
    this.options       = $.extend({}, Collapse.DEFAULTS, options)
    this.$trigger      = $('[data-toggle="collapse"][href="#' + element.id + '"],' +
                           '[data-toggle="collapse"][data-target="#' + element.id + '"]')
    this.transitioning = null

    if (this.options.parent) {
      this.$parent = this.getParent()
    } else {
      this.addAriaAndCollapsedClass(this.$element, this.$trigger)
    }

    if (this.options.toggle) this.toggle()
  }

  Collapse.VERSION  = '3.3.7'

  Collapse.TRANSITION_DURATION = 350

  Collapse.DEFAULTS = {
    toggle: true
  }

  Collapse.prototype.dimension = function () {
    var hasWidth = this.$element.hasClass('width')
    return hasWidth ? 'width' : 'height'
  }

  Collapse.prototype.show = function () {
    if (this.transitioning || this.$element.hasClass('in')) return

    var activesData
    var actives = this.$parent && this.$parent.children('.panel').children('.in, .collapsing')

    if (actives && actives.length) {
      activesData = actives.data('bs.collapse')
      if (activesData && activesData.transitioning) return
    }

    var startEvent = $.Event('show.bs.collapse')
    this.$element.trigger(startEvent)
    if (startEvent.isDefaultPrevented()) return

    if (actives && actives.length) {
      Plugin.call(actives, 'hide')
      activesData || actives.data('bs.collapse', null)
    }

    var dimension = this.dimension()

    this.$element
      .removeClass('collapse')
      .addClass('collapsing')[dimension](0)
      .attr('aria-expanded', true)

    this.$trigger
      .removeClass('collapsed')
      .attr('aria-expanded', true)

    this.transitioning = 1

    var complete = function () {
      this.$element
        .removeClass('collapsing')
        .addClass('collapse in')[dimension]('')
      this.transitioning = 0
      this.$element
        .trigger('shown.bs.collapse')
    }

    if (!$.support.transition) return complete.call(this)

    var scrollSize = $.camelCase(['scroll', dimension].join('-'))

    this.$element
      .one('bsTransitionEnd', $.proxy(complete, this))
      .emulateTransitionEnd(Collapse.TRANSITION_DURATION)[dimension](this.$element[0][scrollSize])
  }

  Collapse.prototype.hide = function () {
    if (this.transitioning || !this.$element.hasClass('in')) return

    var startEvent = $.Event('hide.bs.collapse')
    this.$element.trigger(startEvent)
    if (startEvent.isDefaultPrevented()) return

    var dimension = this.dimension()

    this.$element[dimension](this.$element[dimension]())[0].offsetHeight

    this.$element
      .addClass('collapsing')
      .removeClass('collapse in')
      .attr('aria-expanded', false)

    this.$trigger
      .addClass('collapsed')
      .attr('aria-expanded', false)

    this.transitioning = 1

    var complete = function () {
      this.transitioning = 0
      this.$element
        .removeClass('collapsing')
        .addClass('collapse')
        .trigger('hidden.bs.collapse')
    }

    if (!$.support.transition) return complete.call(this)

    this.$element
      [dimension](0)
      .one('bsTransitionEnd', $.proxy(complete, this))
      .emulateTransitionEnd(Collapse.TRANSITION_DURATION)
  }

  Collapse.prototype.toggle = function () {
    this[this.$element.hasClass('in') ? 'hide' : 'show']()
  }

  Collapse.prototype.getParent = function () {
    return $(this.options.parent)
      .find('[data-toggle="collapse"][data-parent="' + this.options.parent + '"]')
      .each($.proxy(function (i, element) {
        var $element = $(element)
        this.addAriaAndCollapsedClass(getTargetFromTrigger($element), $element)
      }, this))
      .end()
  }

  Collapse.prototype.addAriaAndCollapsedClass = function ($element, $trigger) {
    var isOpen = $element.hasClass('in')

    $element.attr('aria-expanded', isOpen)
    $trigger
      .toggleClass('collapsed', !isOpen)
      .attr('aria-expanded', isOpen)
  }

  function getTargetFromTrigger($trigger) {
    var href
    var target = $trigger.attr('data-target')
      || (href = $trigger.attr('href')) && href.replace(/.*(?=#[^\s]+$)/, '') // strip for ie7

    return $(target)
  }


  // COLLAPSE PLUGIN DEFINITION
  // ==========================

  function Plugin(option) {
    return this.each(function () {
      var $this   = $(this)
      var data    = $this.data('bs.collapse')
      var options = $.extend({}, Collapse.DEFAULTS, $this.data(), typeof option == 'object' && option)

      if (!data && options.toggle && /show|hide/.test(option)) options.toggle = false
      if (!data) $this.data('bs.collapse', (data = new Collapse(this, options)))
      if (typeof option == 'string') data[option]()
    })
  }

  var old = $.fn.collapse

  $.fn.collapse             = Plugin
  $.fn.collapse.Constructor = Collapse


  // COLLAPSE NO CONFLICT
  // ====================

  $.fn.collapse.noConflict = function () {
    $.fn.collapse = old
    return this
  }


  // COLLAPSE DATA-API
  // =================

  $(document).on('click.bs.collapse.data-api', '[data-toggle="collapse"]', function (e) {
    var $this   = $(this)

    if (!$this.attr('data-target')) e.preventDefault()

    var $target = getTargetFromTrigger($this)
    var data    = $target.data('bs.collapse')
    var option  = data ? 'toggle' : $this.data()

    Plugin.call($target, option)
  })

}(jQuery);
</script>
<script>
window.initializeCodeFolding = function(show) {

  // handlers for show-all and hide all
  $("#rmd-show-all-code").click(function() {
    $('div.r-code-collapse').each(function() {
      $(this).collapse('show');
    });
  });
  $("#rmd-hide-all-code").click(function() {
    $('div.r-code-collapse').each(function() {
      $(this).collapse('hide');
    });
  });
  
  // index for unique code element ids
  var currentIndex = 1;

  // select all R code blocks
  var rCodeBlocks = $('pre.sourceCode, pre.r, pre.python, pre.bash, pre.sql, pre.cpp, pre.stan');
  rCodeBlocks.each(function() {

    // create a collapsable div to wrap the code in
    var div = $('<div class="collapse r-code-collapse"></div>');
    if (show)
      div.addClass('in');
    var id = 'rcode-643E0F36' + currentIndex++;
    div.attr('id', id);
    $(this).before(div);
    $(this).detach().appendTo(div);

    // add a show code button right above
    // var showCodeText = $('<span>' + (show ? 'Hide' : 'Code') + '</span>');
    var showCodeText = $('<img style="width:45px;" src="img/both.svg">');
    var showCodeButton = $('<button type="button" class="btn btn-default btn-xs code-folding-btn pull-right"></button>');
    showCodeButton.append(showCodeText);
    showCodeButton
        .attr('data-toggle', 'collapse')
        .attr('data-target', '#' + id)
        .attr('aria-expanded', show)
        .attr('aria-controls', id);

    var buttonRow = $('<div class="row"></div>');
    var buttonCol = $('<div class="col-md-12"></div>');

    var buttonColK = $('<div class="check"><input type="checkbox" id="cc_' + id + '" class="btn btn-default btn-xs code-folding-btn pull-right ch1" /><label class="checktrue" for="cc_' + id + '" ></label></div>');

    buttonColK
        .attr('data-toggle', 'collapse')
        .attr('data-target', '#' + id)
        .attr('aria-expanded', show)
        .attr('aria-controls', id);

    //buttonCol.append(showCodeButton);
    buttonRow.append(buttonCol);
    buttonCol.append(buttonColK);

    div.before(buttonRow);

    // update state of button on show/hide
    div.on('hidden.bs.collapse', function () {
      showCodeText.text();
    });
    div.on('show.bs.collapse', function () {
      showCodeText.text();
    });
  });

}
</script>
<script>
/* ========================================================================
 * Bootstrap: dropdown.js v3.3.7
 * http://getbootstrap.com/javascript/#dropdowns
 * ========================================================================
 * Copyright 2011-2016 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 * ======================================================================== */


+function ($) {
  'use strict';

  // DROPDOWN CLASS DEFINITION
  // =========================

  var backdrop = '.dropdown-backdrop'
  var toggle   = '[data-toggle="dropdown"]'
  var Dropdown = function (element) {
    $(element).on('click.bs.dropdown', this.toggle)
  }

  Dropdown.VERSION = '3.3.7'

  function getParent($this) {
    var selector = $this.attr('data-target')

    if (!selector) {
      selector = $this.attr('href')
      selector = selector && /#[A-Za-z]/.test(selector) && selector.replace(/.*(?=#[^\s]*$)/, '') // strip for ie7
    }

    var $parent = selector && $(selector)

    return $parent && $parent.length ? $parent : $this.parent()
  }

  function clearMenus(e) {
    if (e && e.which === 3) return
    $(backdrop).remove()
    $(toggle).each(function () {
      var $this         = $(this)
      var $parent       = getParent($this)
      var relatedTarget = { relatedTarget: this }

      if (!$parent.hasClass('open')) return

      if (e && e.type == 'click' && /input|textarea/i.test(e.target.tagName) && $.contains($parent[0], e.target)) return

      $parent.trigger(e = $.Event('hide.bs.dropdown', relatedTarget))

      if (e.isDefaultPrevented()) return

      $this.attr('aria-expanded', 'false')
      $parent.removeClass('open').trigger($.Event('hidden.bs.dropdown', relatedTarget))
    })
  }

  Dropdown.prototype.toggle = function (e) {
    var $this = $(this)

    if ($this.is('.disabled, :disabled')) return

    var $parent  = getParent($this)
    var isActive = $parent.hasClass('open')

    clearMenus()

    if (!isActive) {
      if ('ontouchstart' in document.documentElement && !$parent.closest('.navbar-nav').length) {
        // if mobile we use a backdrop because click events don't delegate
        $(document.createElement('div'))
          .addClass('dropdown-backdrop')
          .insertAfter($(this))
          .on('click', clearMenus)
      }

      var relatedTarget = { relatedTarget: this }
      $parent.trigger(e = $.Event('show.bs.dropdown', relatedTarget))

      if (e.isDefaultPrevented()) return

      $this
        .trigger('focus')
        .attr('aria-expanded', 'true')

      $parent
        .toggleClass('open')
        .trigger($.Event('shown.bs.dropdown', relatedTarget))
    }

    return false
  }

  Dropdown.prototype.keydown = function (e) {
    if (!/(38|40|27|32)/.test(e.which) || /input|textarea/i.test(e.target.tagName)) return

    var $this = $(this)

    e.preventDefault()
    e.stopPropagation()

    if ($this.is('.disabled, :disabled')) return

    var $parent  = getParent($this)
    var isActive = $parent.hasClass('open')

    if (!isActive && e.which != 27 || isActive && e.which == 27) {
      if (e.which == 27) $parent.find(toggle).trigger('focus')
      return $this.trigger('click')
    }

    var desc = ' li:not(.disabled):visible a'
    var $items = $parent.find('.dropdown-menu' + desc)

    if (!$items.length) return

    var index = $items.index(e.target)

    if (e.which == 38 && index > 0)                 index--         // up
    if (e.which == 40 && index < $items.length - 1) index++         // down
    if (!~index)                                    index = 0

    $items.eq(index).trigger('focus')
  }


  // DROPDOWN PLUGIN DEFINITION
  // ==========================

  function Plugin(option) {
    return this.each(function () {
      var $this = $(this)
      var data  = $this.data('bs.dropdown')

      if (!data) $this.data('bs.dropdown', (data = new Dropdown(this)))
      if (typeof option == 'string') data[option].call($this)
    })
  }

  var old = $.fn.dropdown

  $.fn.dropdown             = Plugin
  $.fn.dropdown.Constructor = Dropdown


  // DROPDOWN NO CONFLICT
  // ====================

  $.fn.dropdown.noConflict = function () {
    $.fn.dropdown = old
    return this
  }


  // APPLY TO STANDARD DROPDOWN ELEMENTS
  // ===================================

  $(document)
    .on('click.bs.dropdown.data-api', clearMenus)
    .on('click.bs.dropdown.data-api', '.dropdown form', function (e) { e.stopPropagation() })
    .on('click.bs.dropdown.data-api', toggle, Dropdown.prototype.toggle)
    .on('keydown.bs.dropdown.data-api', toggle, Dropdown.prototype.keydown)
    .on('keydown.bs.dropdown.data-api', '.dropdown-menu', Dropdown.prototype.keydown)

}(jQuery);
</script>
<script>
$(document).ready(function() {

  // Section anchors
  $('.section h1, .section h2, .section h3, .section h4, .section h5').each(function() {
    anchor = '#' + $(this).parent().attr('id');
    $(this).addClass("hasAnchor").append('<a href="' + anchor + '" class="anchor"></a>');
  });
});
</script>
<link rel="shortcut icon" href="img/favicon.ico" />
<script>
$(document).ready(function () {
  window.initializeCodeFolding("show" === "show");
});
</script>

<script>document.write('<div class="btn-group pull-right" style="position: absolute; top: 35px; right: 2%; z-index: 10"><button type="button" class="btn btn-default btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"aria-expanded="true" data-_extension-text-contrast=""><span>global code</span><span class="caret"></span></button><ul class="dropdown-menu" style="min-width: 50px;"><li><a id="rmd-show-all-code" href="#">Show All</a></li><li><a id="rmd-hide-all-code" href="#">Hide All</a></li></ul></div>')</script>


<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(title);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="https://giselakopp.netlify.app/">Sociality & Evolution</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Intro</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#dependencies"><i class="fa fa-check"></i><b>1.1</b> Dependencies</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="project-summary.html"><a href="project-summary.html"><i class="fa fa-check"></i><b>2</b> Project summary</a><ul>
<li class="chapter" data-level="2.1" data-path="project-summary.html"><a href="project-summary.html#general-idea"><i class="fa fa-check"></i><b>2.1</b> General idea</a></li>
<li class="chapter" data-level="2.2" data-path="project-summary.html"><a href="project-summary.html#project-overview"><i class="fa fa-check"></i><b>2.2</b> Project overview</a><ul>
<li class="chapter" data-level="2.2.1" data-path="project-summary.html"><a href="project-summary.html#work-plan"><i class="fa fa-check"></i><b>2.2.1</b> Work plan</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="project-summary.html"><a href="project-summary.html#project-metadata"><i class="fa fa-check"></i><b>2.3</b> Project metadata</a><ul>
<li class="chapter" data-level="2.3.1" data-path="project-summary.html"><a href="project-summary.html#species"><i class="fa fa-check"></i><b>2.3.1</b> Species</a></li>
<li class="chapter" data-level="2.3.2" data-path="project-summary.html"><a href="project-summary.html#locations"><i class="fa fa-check"></i><b>2.3.2</b> Locations</a></li>
<li class="chapter" data-level="2.3.3" data-path="project-summary.html"><a href="project-summary.html#dates"><i class="fa fa-check"></i><b>2.3.3</b> Dates</a></li>
<li class="chapter" data-level="2.3.4" data-path="project-summary.html"><a href="project-summary.html#people"><i class="fa fa-check"></i><b>2.3.4</b> People</a></li>
<li class="chapter" data-level="2.3.5" data-path="project-summary.html"><a href="project-summary.html#permits"><i class="fa fa-check"></i><b>2.3.5</b> Permits</a></li>
<li class="chapter" data-level="2.3.6" data-path="project-summary.html"><a href="project-summary.html#funding"><i class="fa fa-check"></i><b>2.3.6</b> Funding</a></li>
<li class="chapter" data-level="2.3.7" data-path="project-summary.html"><a href="project-summary.html#data-storage"><i class="fa fa-check"></i><b>2.3.7</b> Data storage</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="project-summary.html"><a href="project-summary.html#sample-collection-protocol"><i class="fa fa-check"></i><b>2.4</b> Sample collection protocol</a><ul>
<li class="chapter" data-level="2.4.1" data-path="project-summary.html"><a href="project-summary.html#equipment"><i class="fa fa-check"></i><b>2.4.1</b> Equipment</a></li>
<li class="chapter" data-level="2.4.2" data-path="project-summary.html"><a href="project-summary.html#conditions-for-samples-in-storage-solution"><i class="fa fa-check"></i><b>2.4.2</b> Conditions for samples in storage solution</a></li>
<li class="chapter" data-level="2.4.3" data-path="project-summary.html"><a href="project-summary.html#collection-of-samples"><i class="fa fa-check"></i><b>2.4.3</b> Collection of samples</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="project-notes.html"><a href="project-notes.html"><i class="fa fa-check"></i><b>3</b> Project notes</a><ul>
<li class="chapter" data-level="3.1" data-path="project-notes.html"><a href="project-notes.html#meeting-notes"><i class="fa fa-check"></i><b>3.1</b> Meeting notes</a></li>
<li class="chapter" data-level="3.2" data-path="project-notes.html"><a href="project-notes.html#reading-notes"><i class="fa fa-check"></i><b>3.2</b> Reading (notes)</a></li>
<li class="chapter" data-level="3.3" data-path="project-notes.html"><a href="project-notes.html#ideas-to-follow-up-on"><i class="fa fa-check"></i><b>3.3</b> Ideas to follow up on</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="data.html"><a href="data.html"><i class="fa fa-check"></i><b>4</b> Data</a><ul>
<li class="chapter" data-level="4.1" data-path="data.html"><a href="data.html#data-collection"><i class="fa fa-check"></i><b>4.1</b> Data collection</a><ul>
<li class="chapter" data-level="4.1.1" data-path="data.html"><a href="data.html#nyctalus-noctula"><i class="fa fa-check"></i><b>4.1.1</b> Nyctalus noctula</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="data.html"><a href="data.html#data-processing"><i class="fa fa-check"></i><b>4.2</b> Data processing</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="analysis.html"><a href="analysis.html"><i class="fa fa-check"></i><b>5</b> Analysis</a><ul>
<li class="chapter" data-level="5.1" data-path="analysis.html"><a href="analysis.html#analysis-1"><i class="fa fa-check"></i><b>5.1</b> Analysis 1</a></li>
<li class="chapter" data-level="5.2" data-path="analysis.html"><a href="analysis.html#analysis-2"><i class="fa fa-check"></i><b>5.2</b> Analysis 2</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="references-and-session-info.html"><a href="references-and-session-info.html"><i class="fa fa-check"></i><b>6</b> References and Session Info</a><ul>
<li class="chapter" data-level="6.1" data-path="references-and-session-info.html"><a href="references-and-session-info.html#session-info"><i class="fa fa-check"></i><b>6.1</b> Session Info</a></li>
<li class="chapter" data-level="6.2" data-path="references-and-session-info.html"><a href="references-and-session-info.html#references"><i class="fa fa-check"></i><b>6.2</b> References</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">fecalFACS Notebook</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="project-summary" class="section level1">
<h1><span class="header-section-number">2</span> Project summary</h1>
<div id="general-idea" class="section level2">
<h2><span class="header-section-number">2.1</span> General idea</h2>
<p>In this project, we will assess the general applicability of the fecalFACS method. We submitted a medium money grant proposal to the CASCB on 13.04.2021. We received the positive answer of funding of our project (informally) on 08.06.2021.</p>
</div>
<div id="project-overview" class="section level2">
<h2><span class="header-section-number">2.2</span> Project overview</h2>
<p>More than five decades ago, W. D. Hamilton formulated his seminal “rule” on why individuals cooperate: They increase their inclusive fitness (i.e. the number of shared genes passed to the next genera-tion) by promoting the survival or reproduction of close kin <span class="citation">(Hamilton <a href="references-and-session-info.html#ref-Hamilton1964-bw" role="doc-biblioref">1964</a><a href="references-and-session-info.html#ref-Hamilton1964-bw" role="doc-biblioref">a</a>, <a href="references-and-session-info.html#ref-Hamilton1964-ne" role="doc-biblioref">1964</a><a href="references-and-session-info.html#ref-Hamilton1964-ne" role="doc-biblioref">b</a>)</span>. The idea of kin selection as a driving force for social interactions has since become a central paradigm in behavioural ecology and evolutionary biology with many studies supporting its predictions; but it is also essential to weigh kin selection against other potential drivers, such as reciprocal altruism or direct benefits <span class="citation">(Smith <a href="references-and-session-info.html#ref-Smith2014-dk" role="doc-biblioref">2014</a>)</span>.
Testing these hypotheses is only possible if relatedness between individuals can be determined, which has been a difficult endeavour in natural populations of difficult-to-sample taxa. Standard methods to generate genetic data rely on DNA of high quality and quantity, which can only be obtained from blood or tissue samples. Collecting these samples from dozens or hundreds of individuals, as desirable for behavioural studies, is often not feasible or may be prohibited. The development of methods that can generate genetic data from non-invasively collected samples (e.g. faeces, hair, and urine) revolutionized the field of behavioural ecology <span class="citation">(Piggott and Taylor <a href="references-and-session-info.html#ref-Piggott2003-tc" role="doc-biblioref">2003</a>)</span>. These methods have traditionally used genetic marker systems that only represent a very small fraction of the genome and are imprecise in resolving fine-scale relationship patterns, especially in wild animal groups <span class="citation">(Van Horn, Altmann, and Alberts <a href="references-and-session-info.html#ref-Van_Horn2008-dx" role="doc-biblioref">2008</a>)</span>. Thus, while being sufficient for basic population genetic studies or pedigree reconstructions in well-known study populations, the techniques for non-invasive genetic analysis often fail to provide enough detail and accuracy in populations without background demographic data. Additionally, they require substantial initial investments in developing appropriate genetic markers for species lacking genomic data.
With novel sequencing technologies <span class="citation">(Metzker <a href="references-and-session-info.html#ref-Metzker2010-kd" role="doc-biblioref">2010</a>)</span>, sequencing of whole genomes has become attainable, and with decreasing costs genomics has become applicable to wild populations <span class="citation">(Ellegren <a href="references-and-session-info.html#ref-Ellegren2014-pu" role="doc-biblioref">2014</a>)</span>. Unlocking population genomics for non-invasively collected samples would make it possible to gather high-resolution relatedness data on wild populations of non-model organisms and to include gene-specific information instead of genome-wide averages. Studies based on whole-genome data can also take advantage of the most recent analytical tool developments and are more comparable across species. Hence, they would allow to elucidate how genotypic composition or functional genomic variation shape the collective behaviour of wild animal groups, research questions that have provided exciting results in laboratory experiments <span class="citation">(Tang et al. <a href="references-and-session-info.html#ref-Tang2020-nu" role="doc-biblioref">2020</a>; Walsh et al. <a href="references-and-session-info.html#ref-Walsh2020-xo" role="doc-biblioref">2020</a>)</span> but are currently not feasible in most natural populations.
Although becoming cheaper, genome sequencing is still very expensive, especially if a large number of individuals is to be sequenced (e.g. all individuals in replicated social groups). One solution is to sequence many individual whole genomes to low coverage <span class="citation">(Nielsen et al. <a href="references-and-session-info.html#ref-Nielsen2011-tz" role="doc-biblioref">2011</a>; Le and Durbin <a href="references-and-session-info.html#ref-Le2011-dd" role="doc-biblioref">2011</a>)</span> and apply probabilistic models across all nucleotide positions to calculate genotype likelihoods instead of trying to guess the “true” genotype per site <span class="citation">(Fuentes-Pardo and Ruzzante <a href="references-and-session-info.html#ref-Fuentes-Pardo2017-ih" role="doc-biblioref">2017</a>)</span>. A number of population genomic methods and analytical tools have now been developed to specifically estimate genomic relatedness from such shallow sequencing data <span class="citation">(Korneliussen and Moltke <a href="references-and-session-info.html#ref-Korneliussen2015-ie" role="doc-biblioref">2015</a>; Monroy Kuhn, Jakobsson, and Günther <a href="references-and-session-info.html#ref-Monroy_Kuhn2018-aa" role="doc-biblioref">2018</a>; Fernandes et al. <a href="references-and-session-info.html#ref-Fernandes2017-ph" role="doc-biblioref">2017</a>; Carpenter et al. <a href="references-and-session-info.html#ref-Carpenter2013-wc" role="doc-biblioref">2013</a>)</span>. While they mainly targeted studies of ancient and prehistoric human remains, they also hold great promise for studies of natural animal populations, especially if samples can only be obtained non-invasively and the resulting DNA holds the same pitfalls as ancient DNA (aDNA), i.e. very low DNA quality and quantity as well as extreme levels of contamination <span class="citation">(Snyder-Mackler et al. <a href="references-and-session-info.html#ref-Snyder-Mackler2016-ki" role="doc-biblioref">2016</a>)</span>.<br />
These advances have huge potential to study kin selection and collective behaviour in the wild, but to date have not widely been integrated in field studies. The hesitancy of the animal behaviour research community to adapt these new approaches can be attributed to the perceived high expertise in molecular technologies and bioinformatics as well as the infrastructure required to successfully implement these protocols, which is usually beyond the capacity of individual research groups. I contributed to the development of one of these methods, which allows the reconstruction of pedigrees in wild animal populations <span class="citation">(Snyder-Mackler et al. <a href="references-and-session-info.html#ref-Snyder-Mackler2016-ki" role="doc-biblioref">2016</a>)</span>. This method estimates relatedness from low coverage whole genome data and avoids sequencing high amounts of contaminant DNA (which can be above 99% in fecal samples) by enriching target DNA with capture baits prior to sequencing. The shortcoming of this method is that it still requires one high quality DNA sample to generate species-specific RNA baits for capture enrichment. This limitation is circumvented in the novel fecalFACS approach <span class="citation">(Orkin et al. <a href="references-and-session-info.html#ref-Orkin2021-he" role="doc-biblioref">2021</a>)</span>, which uses fluorescence-activated cell sorting to preselect target cells from which DNA is subsequently extracted and sequenced.
In this project, we will test the fecalFACS method as a tool to study genomic relatedness networks that underlie social interaction networks in natural animal populations. Importantly, we aim to implement a low-threshold semiautomatic pipeline, in which researchers can analyse their samples for genomic relatedness with little prior expert knowledge in the applied methods.</p>
<p>We will use two primate and two bat species as study systems to investigate the broad-scale feasibility and accuracy of the fecalFACS method. One primate and one bat species will serve as “setup spe-cies” to establish the method, the remaining two species will serve as “verification species” to evaluate the generalizability of the established workflow. The study species are selected for three main reasons: First, both primates and bats live in complex societies with highly differentiated social relationships, making them prominent study systems to investigate social interaction networks and the evolution of cooperative behaviours. However, while kin selection is invoked as a strong factor in shaping social relationships in many group-living primates <span class="citation">(Silk <a href="references-and-session-info.html#ref-Silk2002-br" role="doc-biblioref">2002</a>)</span>, its role in bats appears much more ambiguous <span class="citation">(Wilkinson et al. <a href="references-and-session-info.html#ref-Wilkinson2019-no" role="doc-biblioref">2019</a>)</span>. This will give us the opportunity to investigate genomic relatedness networks in species with contrasting predictions: we predict that in the primate species, social interaction networks reflect genomic relatedness networks, whereas we predict to find no strong link for the bat species. Second, we need to consider the influence of diet on the tested method. A high proportion of mammalian prey items in the diet of the study species would likely decrease the amount of target sequencing reads, because we would expect that many of the selected cells from the flow cytometry step would not represent the target study species but the prey it consumed. While we therefore chose to not investigate carnivorous species, we chose species with different diets that permit us to evaluate whether faunal prey can potentially be analysed simultaneously to the genomic relatedness networks. This could make this method especially interesting for species with cooperative hunting, as it would allow us to determine the individuals that consumed the same prey during a given time window, as expected for cooperative hunting, including in both target bat species. Third, we selected study populations where we have ongoing studies focusing on collective behaviour. For these populations, social interaction data are already available, but relatedness data would be crucial to interpret evolutionary questions. The collection of fecal samples is highly feasible in a short-term time frame and project delay due to time-consuming issuing of necessary permissions or the Covid19 pandemic can be ruled out (“setup species”) or minimized (“verification species”).</p>
<div id="work-plan" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Work plan</h3>
<p>Study groups and sample collection: Faecal samples will be collected from at least 40 individuals for each of the four study species. Samples from the “primate setup species” Barbary macaques (<em>Macaca sylvanus</em>) will be collected at Affenberg Salem, Germany, where approx. 180 Barbary macaques live in three groups under semi-natural conditions in a 14.5 ha forest enclosure. Samples from the “primate verification species” Ugandan red colobus (<em>Piliocolobus tephrosceles</em>) will be collected by Urs Kalbitzer and his team at the long-term research project at Kanyawara in Kibale National Park, Uganda. During daily observational follows, a sample will be collected immediately after defecation by an identified individual. Samples from the “bat setup species”, the common noctule bat (<em>Nyctalus noctula</em>) will be collected by Dina Dechmann and her team at the MPI-AB study sites in the Konstanz region. Samples from the “bat verification species”, the Greater noctule bat (<em>Nyctalus lasiopterus</em>) will be collected by Dina Dechmann and her collaborators Carlos Ibanez and Javier Juste at the Estación Biológica de Doñana (CSIC) in Spain. Samples will be collected from roosts during regular monitoring visits and identities of all individuals within the roost will be recorded. All samples will be stored in 5ml RNAlater at ambient temperature until transport to the laboratory at the University of Konstanz.</p>
<div id="behavioural-data" class="section level4">
<h4><span class="header-section-number">2.2.1.1</span> Behavioural data</h4>
<p>Association and social interaction data will be contributed by the collaborators of the respective study species and used to construct social interaction networks using the igraph package <span class="citation">(Csardi and Nepusz <a href="references-and-session-info.html#ref-Csardi2006-bs" role="doc-biblioref">2006</a>)</span> in R <span class="citation">(R Development Core Team <a href="references-and-session-info.html#ref-R_Development_Core_Team2020-vb" role="doc-biblioref">2020</a>)</span>. For Barbary macaques, we will use data on group and matrilineal membership, male coalitions, and male-infant interactions. For red colobus, we will use data on group member-ship, proximity, and grooming interactions. For both noctule bat species, we will use roosting association data as well as telemetry or GPS tracking data of groups.</p>
</div>
<div id="fecalfacs-and-sequencing" class="section level4">
<h4><span class="header-section-number">2.2.1.2</span> FecalFACS and Sequencing</h4>
<p>Target cells for sequencing will be isolated from the fecal samples on a BD FACSAria Fusion (BD Biosciences) flow cytometer at the Flow Cytometry Centre (FlowKon) at the University of Konstanz following the published fecalFACS protocol <span class="citation">(Orkin et al. <a href="references-and-session-info.html#ref-Orkin2021-he" role="doc-biblioref">2021</a>)</span>. We will implement slight modifications to increase the precision of the cell sorting, i.e. evaluate the combination of different stains to achieve more specific selection criteria for each species. The effectiveness of the cell sorting will be microscopically confirmed at the BioImaging Centre (BIC) at the University of Konstanz.
DNA laboratory work will be conducted at the University of Konstanz with necessary precautions for the handling of fecal DNA. DNA will be extracted from the isolated cells using the Qiagen DNA Micro kit with modifications to increase DNA yield <span class="citation">(Orkin et al. <a href="references-and-session-info.html#ref-Orkin2021-he" role="doc-biblioref">2021</a>)</span>. Final DNA concentration will be measured with a Qubit fluorometer and quality checked on a TapeStation 2200. Whole-genome sequencing libraries will be prepared with the NEBNext Ultra II kit and 150bp paired-end sequenced on an Illumina NovaSeq6000 at the Sequencing Core facility of the MPI Molecular Genetics in Berlin to achieve 1-2X coverage.</p>
</div>
<div id="sequencing-data-processing-and-analyses" class="section level4">
<h4><span class="header-section-number">2.2.1.3</span> Sequencing data processing and analyses</h4>
<p>Sequence data will be processed and analysed with a modified version of our nextflow pipeline for genotype likelihood calling and relatedness estimation from shallow whole genome sequencing data, which we have developed in collaboration with the Sequencing Analysis Core Facility (SequAna) at the University of Konstanz. We will use the <em>M. sylvanus</em> data set to evaluate the precision and accuracy of our analyses by comparing the obtained relatedness coefficients to long-term monitoring data of the study population and known mother-offspring pairs (for which a relatedness coefficient of 0.5 is to be expected). For each species, we will build relatedness networks using relatedness coefficients as edge weights using igraph. To assess the correlation between social interaction networks and genomic relatedness networks we will use multilayer network methods <span class="citation">(Finn et al. <a href="references-and-session-info.html#ref-Finn2019-ko" role="doc-biblioref">2019</a>)</span> and (for each species) calculate edge overlap between the constructed networks as implemented in MuxViz <span class="citation">(De Domenico, Porter, and Arenas <a href="references-and-session-info.html#ref-De_Domenico2014-lw" role="doc-biblioref">2014</a>)</span>.</p>
</div>
</div>
</div>
<div id="project-metadata" class="section level2">
<h2><span class="header-section-number">2.3</span> Project metadata</h2>
<div id="species" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Species</h3>
<ul>
<li>Barbary macaque (<em>Macaca sylvanus</em>)<br />
</li>
<li>Ugandan red colobus (<em>Piliocolobus tephrosceles</em>)<br />
</li>
<li>common noctule bat (<em>Nyctalus noctula</em>)<br />
</li>
<li>Greater noctule bat (<em>Nyctalus lasiopterus</em>)</li>
</ul>
</div>
<div id="locations" class="section level3">
<h3><span class="header-section-number">2.3.2</span> Locations</h3>
<p>Samples were collected in the following locations:</p>
<ul>
<li><p><strong>Affenberg Salem</strong> (Barbary macaque samples)<br />
Salem, Germany<br />
Contact: Dr. Roland Hilgartner (director)<br />
responsible: Gisela</p></li>
<li><p><strong>Kanyawara</strong> (red colobus)<br />
Uganda<br />
Kibale National Park<br />
responsible: Urs</p></li>
<li><p><strong>Konstanz</strong> (common noctule)<br />
Germany<br />
responsible: Dina</p></li>
<li><p><strong>Estacion Biologica de Donana (CSIC)</strong> (Greater noctule)<br />
Spain<br />
contact: Javier Juste<br />
responsible: Dina</p></li>
</ul>
</div>
<div id="dates" class="section level3">
<h3><span class="header-section-number">2.3.3</span> Dates</h3>
<p>Samples were collected:</p>
<ul>
<li>04/2021 (common noctule bat, Konstanz/Kreuzlingen)</li>
<li>05- 2021 (Barbary macaques, Affenberg Salem)</li>
<li>07/2021 (Greater noctule, Donana)</li>
</ul>
</div>
<div id="people" class="section level3">
<h3><span class="header-section-number">2.3.4</span> People</h3>
<p><strong>Gisela H. Kopp</strong>
PI<br />
Sociality &amp; Evolution/Zukunftskolleg &amp; Dept. Biology, University of Konstanz<br />
<a href="mailto:gisela.kopp@uni-konstanz.de" class="email">gisela.kopp@uni-konstanz.de</a><br />
responsibilities: general project coordination, finances, sample collection Barbary macaques, laboratory analysis and supervision of research assistant, data analysis, manuscript writing</p>
<p><strong>Dina Dechmann</strong>
Co-PI<br />
Migration/MPI Animal Behaviour<br />
<a href="mailto:ddechmann@ab.mpg.de" class="email">ddechmann@ab.mpg.de</a><br />
responsibilities: sample collection bats (incl. permits), bat behavioural data, manuscript writing</p>
<p><strong>Urs Kalbitzer</strong>
Co-PI<br />
EAS/MPI Animal Behaviour<br />
<a href="mailto:ukalbitzer@ab.mpg.de" class="email">ukalbitzer@ab.mpg.de</a><br />
responsibilities: sample collection colobus (incl. permits), colobus behavioural data, manuscript writing</p>
<p><strong>Camila Calderon</strong>
Co-PI<br />
Migration/MPI Animal Behaviour<br />
<a href="mailto:ccalderon@ab.mpg.de" class="email">ccalderon@ab.mpg.de</a><br />
responsibilities: <em>P. discolor</em> sample collection (incl. permits)</p>
<p><strong>Melina Dietzer</strong>
research assistant (until 30/6/2022)<br />
Sociality &amp; Evolution/University of Konstanz<br />
<a href="mailto:mdietzer@ab.mpg.de" class="email">mdietzer@ab.mpg.de</a><br />
responsibilities: laboratory analysis, sample &amp; data management</p>
<p><strong>Javier Juste</strong>
collaborator<br />
Estacion Biologica de Donana (CSIC)
Avda. Americo Vespucio s/n
Seville 41092
Spain
<a href="mailto:juste@ebd.csic.es" class="email">juste@ebd.csic.es</a>
responsibilities: sample collection <em>N. lasiopterus</em> (incl. permits)</p>
<p><strong>Riya Mathur</strong>
research assistant (from 6/2022)<br />
Sociality &amp; Evolution/University of Konstanz<br />
<a href="mailto:riya.mathur@uni-konstanz.de" class="email">riya.mathur@uni-konstanz.de</a><br />
responsibilities: laboratory analysis, sample &amp; data management (until 7/2022)</p>
<p><strong>Amanda Melin</strong>
advice<br />
University of Calgary<br />
<a href="mailto:amanda.melin@ucalgary.ca" class="email">amanda.melin@ucalgary.ca</a><br />
responsibilities: NA</p>
<p><strong>Gwen Duytschaever</strong>
advice<br />
University of Calgary<br />
<a href="mailto:gwen.duytschaever2@ucalgary.ca" class="email">gwen.duytschaever2@ucalgary.ca</a><br />
responsibilities: NA</p>
</div>
<div id="permits" class="section level3">
<h3><span class="header-section-number">2.3.5</span> Permits</h3>
<div class="beware">
<p>Permit name, granting organization, permit number</p>
</div>
</div>
<div id="funding" class="section level3">
<h3><span class="header-section-number">2.3.6</span> Funding</h3>
<p>Project funding via CASCB Medium money grant 2021, CASCB small project grant and Zukunftskolleg co-funding. GHK funded by Hector Pioneer Fellowship (Hector Foundation II &amp; University of Konstanz). DD ???. UK ???. CC ???.</p>
</div>
<div id="data-storage" class="section level3">
<h3><span class="header-section-number">2.3.7</span> Data storage</h3>
</div>
</div>
<div id="sample-collection-protocol" class="section level2">
<h2><span class="header-section-number">2.4</span> Sample collection protocol</h2>
<div id="equipment" class="section level3">
<h3><span class="header-section-number">2.4.1</span> Equipment</h3>
<ul>
<li>Collection tubes 15ml (Eppendorf EP0030122160 €176,40)<br />
</li>
<li>Storage solution:
<ul>
<li>RNAlater Stabilization Solution (Thermo Fisher #AM7021 500ml €580)<br />
or<br />
</li>
<li><del>DNA/RNA shield (ZymoResearch #R1100-250 250ml €241)</del><br />
[alternative: DNA/RNA Fecal Collection Tube Zymo Research #R1101 €108 (10 pack)]</li>
</ul></li>
<li>Tube labels (Carl Roth Cryo-Tags #L673.1 <a href="https://www.carlroth.com/de/de/etiketten/kryo-etiketten-auf-der-rolle-weiss/p/l673.1" class="uri">https://www.carlroth.com/de/de/etiketten/kryo-etiketten-auf-der-rolle-weiss/p/l673.1</a>)<br />
</li>
<li>Collection spatula<br />
</li>
<li>Sample boxes<br />
</li>
<li>Sample collection data sheet<br />
</li>
<li>Plastic gloves</li>
</ul>
<p>Prefill collection tubes with storage solution (<del>9ml DNA/RNA shield or</del> 5ml RNAlater)</p>
</div>
<div id="conditions-for-samples-in-storage-solution" class="section level3">
<h3><span class="header-section-number">2.4.2</span> Conditions for samples in storage solution</h3>
<ul>
<li>DNA/RNA shield: -20°C indefinite; 4-25°C minimum 30 days; 35-40°C 7days<br />
</li>
<li>RNAlater: -20°C indefinite; 4°C 1 month; 25°C 7 days; 35-40°C 1days</li>
</ul>
</div>
<div id="collection-of-samples" class="section level3">
<h3><span class="header-section-number">2.4.3</span> Collection of samples</h3>
<ul>
<li>Locate faecal material. Verify that there is no mixture of samples from different animals.<br />
</li>
<li>Wear gloves and face mask during sample collection<br />
</li>
<li>Place one drop / one spatula of faecal material (ca. 1ml) in collection tube.<br />
</li>
<li>Make sure that faecal material is well covered by storage solution<br />
</li>
<li>Note sample data on tube label and in sample collection data sheet:
<ul>
<li>Material type (i.e. “Faeces”) &amp; storage solution (i.e. “RNAlater” <del>or “DNA/RNA shield”</del>)</li>
<li>Species<br />
</li>
<li>Sample number<br />
</li>
<li>Date, Time<br />
</li>
<li>Collector<br />
</li>
<li>Animal ID (if applicable)<br />
</li>
<li>Group ID (if applicable)<br />
</li>
<li>GPS coordinates (if applicable)<br />
</li>
</ul></li>
<li>Note sample number on tube cap<br />
</li>
<li>Place sample tubes in sample boxes<br />
</li>
<li>Store sample at ambient temperature or in fridge upon shipment to the laboratory</li>
</ul>
<hr />

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="project-notes.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": "github"
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
